x/orderbook/handler_test.go

```diff
@@ -22,17 +22,14 @@ func TestCreateOrderbook(t *testing.T) {
 	meta := &weave.Metadata{Schema: 1}
 
 	market := &Market{
-		// TODO: handle this in the bucket?
-		Metadata: meta,
-		ID:       weavetest.SequenceID(1),
-		Name:     "Main",
-		Owner:    perm.Address(),
+		ID:    weavetest.SequenceID(1),
```

Comment:
Can you explain what did we assume about migrations. And why comment out?

Reply:
We are not using a migration aware bucket.
This is impossible outside of the migrations package and I have an open pr on weave to enable this. 

Handlers are migration aware, so we use those fields here. Assume empty schema in all bucket objects for now